test/widgets/action_sheet_test.dart

```diff
@@ -553,6 +553,21 @@ void main() {
     });
   });
 
+
+  group('narrowLink URL generation tests', () {
+
+    testWidgets('Full Url Testing', (tester) async {
+      final message = eg.streamMessage();
+      final narrow = TopicNarrow.ofMessage(message);
+      await setupToMessageActionSheet(tester, message: message, narrow: narrow);
```

Comment:
For this test, it matters for us that the `store.realmUrl` used does not end with a slash. However, the default value is "https://chat.example/" (see `eg.account`). We should find a way to override that when setting up the store.